packages/trader/src/AppV2/Components/AccumulatorStats/accumulator-stats-description.tsx

```diff
@@ -3,10 +3,22 @@ import { ActionSheet, Text } from '@deriv-com/quill-ui';
 import { Localize } from '@deriv/translations';
 
 const AccumulatorStatsDescription = ({ onActionSheetClose }: { onActionSheetClose: () => void }) => {
+    const src = 'cfd0bd25af245c1c987106a247cbb81a';
```

Comment:
hey, just help me in understanding, don't we use video-config.ts to store the id's and then use Stream from  @cloudflare/stream-react

Reply:
Hello , yup I added it in video-config.ts . Stream in @cloudflare/stream-react is not helping me out here as it is not working as expected with params I am passing . So I moved to the solution mentioned by the cloudflare official site
https://developers.cloudflare.com/stream/viewing-videos/using-the-stream-player/